src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartDetailAdmin.cshtml

```diff
@@ -54,7 +54,7 @@ else
         <div class="card bg-primary mb-3 position-sticky action-bar">
             <div class="card-body">
                 <div class="row gx-3">
-                    <div class="col">
+                    <div class="col" style="min-width: 40%;">
```

Comment:
Nah

Reply:
Do you know why I added it?

Reply:
Without the reason, try to avoid adding inline style as much as you can